storage/mysql/log_storage.go

```diff
@@ -67,7 +67,10 @@ const (
 			WHERE l.LeafIdentityHash = s.LeafIdentityHash
 			AND s.MerkleLeafHash IN (` + placeholderSQL + `) AND l.TreeId = ? AND s.TreeId = l.TreeId`
 	// TODO(drysdale): this assumes hash size of 32
-	dummyMerkleLeafHash               = "12345678901234567890123456789012"
+	dummyMerkleLeafHash = "00000000000000000000000000000000"
+	// This statement returns a dummy Merkle leaf hash value (which must be
```

Comment:
FWIW, I'd rather write a separate method or change/refactor getLeavesByHashInternal instead of using a dummy value. (Feel free to push back.)